docs/tutorials/smart-contracts/hooks/using-on-key-purchase-hook-to-password-protect.md

```diff
@@ -1,25 +1,29 @@
 ---
-title: Creating a hook to password-protect purchases
```

Comment:
Imo the goal of tutorials is to show people how to build something... And the opposite important part is the "how" but the thing...